src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
```

Comment:
```suggestion
  font-family: "Roboto", Arial, sans-serif;
```
![image](https://user-images.githubusercontent.com/99331793/200809317-f7b62abf-4dcb-4d0d-9d44-ca64a612c706.png)